addons/base_import/models/base_import.py

```diff
@@ -299,19 +299,64 @@ def get_fields_tree(self, model, depth=FIELDS_RECURSION_LIMIT):
             'required': False,
             'fields': [],
             'type': 'id',
+            'model_name': model,
         }]
         if not depth:
             return importable_fields
 
-        model_fields = Model.fields_get()
+        model_fields = Model.fields_get(attributes=[
+            'string', 'required', 'type', 'readonly', 'relation',
+            'definition_record', 'definition_record_field',
+        ])
         blacklist = models.MAGIC_COLUMNS
+
+        for name, field in dict(model_fields).items():
+            if field['type'] not in 'properties':
+                continue
+            definition_record = field['definition_record']
+            definition_record_field = field['definition_record_field']
+
+            target_model = Model.env[Model._fields[definition_record].comodel_name]
+            # TODO: Do we want archive record or not ? Maybe it should depends of the
+            # search view at this state but it seems complicated. Let the caller choose.
```

Comment:
```suggestion
            # TODO: Do we want archive record or not? Maybe it should depend of the
            # search view at this state but it seems complicated. Let the caller choose.
```
Either the caller should choose or we impose a default, please don't depend on views. :)